etc/apache2/includes/instance-chat-standalone.include.template

```diff
@@ -106,8 +108,7 @@
     ProxyPassReverse        /http-bind           http://localhost:${DEPLOYMENT_JITSI_WEB_HTTP_PORT}/http-bind
     ProxyPass               /xmpp-websocket      http://localhost:${DEPLOYMENT_JITSI_WEB_HTTP_PORT}/xmpp-websocket
     ProxyPassReverse        /xmpp-websocket      http://localhost:${DEPLOYMENT_JITSI_WEB_HTTP_PORT}/xmpp-websocket
-    
-    # ACC-90: need to force disablereuse and flushpackets to avoid WebSocket interferences
+    s    # ACC-90: need to force disablereuse and flushpackets to avoid WebSocket interferences
```

Comment:
```suggestion
    # ACC-90: need to force disablereuse and flushpackets to avoid WebSocket interferences
```

Reply:
I haven't understand your suggestion, you deleted the empty line?

Reply:
There is a trailing "s", i have deleted